frontend/src/Router.tsx

```diff
@@ -31,6 +32,10 @@ const router = createBrowserRouter([
         path: "/AddTour",
         element: <AddTour />,
       },
+      {
+        path: "/Tours",
```

Comment:
paths usually start with lowercase. Please make sure all of your routes are lowercased.